components/ReferenceManager/references/ReferencesContainer.tsx

```diff
@@ -232,6 +231,23 @@ function ReferencesContainer({
     setLoading(false);
   };
 
+  useEffectFetchOrgProjects({
```

Comment:
The functionality of these is good but as far as naming goes, I think we can omit the "Effect" and "Fetch" portions and just name: `useOrgProjects`.

This is common pattern in other packages and removes the need to know that the hook uses a "useEffect" or "fetch" behind the scenes. From the component using it's perspective, all that is needed to know is that the hook can be used to get projects somehow. 

Not a release blocker, just a suggestion.

Reply:
Will keep that in mind. Thanks for the tip!